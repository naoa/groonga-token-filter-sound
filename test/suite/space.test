plugin_register token_filters/sound

tokenize TokenBigram \
  "ai ue" \
  --token_filters TokenFilterSound
