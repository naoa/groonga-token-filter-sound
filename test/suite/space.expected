plugin_register token_filters/sound
[[0,0.0,0.0],true]
tokenize TokenBigram   "ai ue"   --token_filters TokenFilterSound
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "ai",
      "position": 0,
      "force_prefix": false
    },
    {
      "value": "i ",
      "position": 1,
      "force_prefix": false
    },
    {
      "value": " u",
      "position": 2,
      "force_prefix": false
    },
    {
      "value": "ue",
      "position": 3,
      "force_prefix": false
    },
    {
      "value": "e",
      "position": 4,
      "force_prefix": false
    }
  ]
]
